docs/components/console/manage-clusters/create-cluster-include.md

```diff
@@ -40,3 +40,7 @@ Users without **Admin** roles can deploy only on `dev`, `test`, or `stage` clust
 ![cluster-healthy](./img/cluster-overview-new-cluster-healthy.png)
 
 3. After the cluster is created, click on the cluster name to visit the cluster detail page.
+
+:::note
+**Cluster auto-pause** is not yet available and only applies to non-Enterprise clusters. Development clusters will be paused if they go unused for two hours. When a cluster is paused, not all functionality will be limited, including the execution of BPMN timers and BPMN message catch events. To resume your cluster, navigate to the **Clusters** tab in Console, select the cluster name, and click **Resume cluster** within the **Cluster Details**.
```

Comment:
>  When a cluster is paused, not all functionality will be limited, including the execution of BPMN timers and BPMN message catch events.

not sure what this means, times will continue to work? ;)

> Development clusters will be paused if they go unused for two hours

i don't think this is how it works

Reply:
I just migrated over the existing content for this PR -- is the content outdated then? Do you know the timeframe of when clusters are paused?

Reply:
@ultraschuppi Do you have any insight here I can update accordingly?

Reply:
the current state of autopausing:

- clusters in trial organisations
- tagged with dev
- will be sent to sleep ~ 8h after they have been created / woken up the last time

we put trial-org clusters to sleep after a few days if we think they are not used (no process instances started), but i don't think this is even communicated

To resolve in a separate PR 👍

Reply:
Also

> When a cluster is paused, not all functionality will be limited, including the execution of BPMN timers and BPMN message catch events.

> not sure what this means, times will continue to work? ;)

I've updated this wording @ultraschuppi 👍